WooCommerce/Classes/Authentication/Epilogue/StoreTableViewCell.swift

```diff
@@ -68,6 +75,13 @@ class StoreTableViewCell: UITableViewCell {
         }
     }
 
+    /// When enabled, the Notice ImageView will be displayed.
+    ///
+    var allowsNotice: Bool = false {
```

Comment:
nit: I think `displays*` matches its functionality more? the previous `allowsCheckmark` is to "allow" all cells for padding, while `displaysCheckmark` is to actually show the checkmark. in the use case here, I think the boolean is to control whether to show the notice icon or not.

```suggestion
    var displaysNotice: Bool = false {
```

Reply:
Updated in d9fed176a87d86d5044c4e778061fa3918493be2.